docs/development-and-code/setup_alternatives/vagrant-setup.md

```diff
@@ -120,12 +120,22 @@ Now that you (hopefully) have a working vagrant machine, you will have to ssh in
 vagrant ssh
 ```
 
+Change to the project directory. This is shared by Vagrant / VirtualBox between your virtual server and your machine for
+easy updating during development:
+
 ```bash
 cd ~/Code/platform-api
 ```
 
+Set required php version. For current version of Ushahidi this should be 7.3:
+
 ```bash
 sudo update-alternatives --set php /usr/bin/php7.3
+sudo systemctl stop php7.1-fpm.service
+sudo systemctl disable php7.1-fpm.service
+sudo systemctl enable php7.3-fpm.service
+sudo systemctl start php7.3-fpm.service
+sudo sed --in-place=.php7.1.bak "s/php7.1-fpm/php7.3-fpm/g" /etc/nginx/sites-available/*
```

Comment:
Looking at this again, I'm kind of sure that I needed to restart nginx in order to activate the changed settings:
```suggestion
sudo sed --in-place=.php7.1.bak "s/php7.1-fpm/php7.3-fpm/g" /etc/nginx/sites-available/*
sudo systemctl restart nginx.service
```